### PR TITLE
Sample metadata/automation file changes from running migration post-process script

### PR DESCRIPTION
**Pull request by vchudnov-g**
This shows the output of running the script in
https://github.com/googleapis/google-cloud-python/pull/10880 using

```
cd scripts/split_repo_migration
./split-repo-post-process.sh ../../../python-speech ../../ google-cloud-speech
```

where `python-speech/` lives in a sibling directory of `google-cloud-python/`.

### CODE REVIEW

File: release-please-config.json [hunk 1]
```diff
@@ -1,177 +1,201 @@
...
-        "packages/google-cloud-alloydb": {
-            "component": "google-cloud-alloydb",
-            "release-type": "python",
-            "extra-files": [
-                "google/cloud/alloydb/gapic_version.py",
-                "google/cloud/alloydb_v1/gapic_version.py",
-                "google/cloud/alloydb_v1alpha/gapic_version.py",
-                "google/cloud/alloydb_v1beta/gapic_version.py"
-            ]
+        "google/cloud/bigquery/biglake_v1alpha1/gapic_version.py",
```
Comment (parthea) [inline release-please-config.json:161]: This entry for `bigquery/biglake` is under `workstations`

Comment (vchudnov-g) [reply release-please-config.json:161]: Yikes! Good catch. I'll look into this.

Comment (vchudnov-g) [reply release-please-config.json:161]: This problem was not created by the migration script. It was introduced earlier in https://github.com/googleapis/google-cloud-python/pull/10861 and I didn't catch it. My other script to release a new library must have an error. I don't this issue should block this migration script, which acted on speech and the speech stuff appears well-configured at the end of the file.

**Comment by parthea** [comment]
Closing as obsolete

**Pull request closed without merging**